config/settings.yml

```diff
@@ -1480,6 +1480,6 @@
       :memory_threshold: 1.gigabytes
       :nice_delta: 1
     :websocket_worker:
-       :connection_pool_size: 8
+       :connection_pool_size: 26
```

Comment:
I agree 5 is too small. But this fix makes me nervous

I thought the workers change the connecting settings or something. Remember something changing configuration behind the scenes because ui has different needs from other types of workers.

I did notice that when I reload code, I run out of connections. Seems the websocket workers eat them. Wonder if we have problems with pages disconnecting.